fava/static/sass/_overlay.scss

```diff
@@ -65,62 +65,31 @@ $overlay-wrapper-background: rgba(0, 0, 0, .5);
       line-height: 1.8em;
     }
   }
-}
-
-.field-control {
-  display: flex;
 
-  input {
-    height: 33px;
+  fieldset {
+    display: flex;
     margin-bottom: .5em;
-    padding: 4px 8px;
-    width: 100%;
+    padding: 0;
 
-    &:last-child {
-      margin-bottom: 0;
+    label {
+      line-height: 33px;
+      width: 150px;
     }
   }
 
-  label {
-    display: inline-block;
-    line-height: 33px;
-    width: 150px;
-  }
-
-  select {
-    font-size: 16px;
-    height: 33px;
-  }
-
-  [type='submit'] {
-    display: block;
-    margin-top: 1rem;
-
-    &:focus {
```

Comment:
The focus states are important IMHO because tabbing through the form and then pressing "enter" when the focus is on the Save is really helpful.